CONTRIBUTING.md

```diff
@@ -1,3 +1,9 @@
+<!--
+SPDX-FileCopyrightText: Ericsson AB
+
+SPDX-License-Identifier: Apache-2.0
```

Comment:
```suggestion
%CopyrightBegin%

SPDX-FileCopyrightText: Ericsson AB 2016-2025. All Rights Reserved.

SPDX-License-Identifier: Apache-2.0

%CopyrightEnd%
```
If the license and copyright information is enclosed in between %CopyrightBegin% and %CopyrightEnd% where we are the original author (like for this file) and the copyright text is formatted as above our script for updating copyright year and changing license information should work.